packages/lodestar/src/chain/interface.ts

```diff
@@ -9,6 +9,7 @@ import {ProgressiveMerkleTree} from "@chainsafe/eth2.0-utils";
 export interface IChainEvents {
   unknownBlockRoot: (root: Hash) => void;
   processedBlock: (block: BeaconBlock) => void;
+  "chain:processedCheckpoint": (checkPoint: Checkpoint) => void;
```

Comment:
can we stick with `processedCheckpoint` for consistency for now? Maybe tackle 'robust-ifying' our event names separately.